doc/faq.md

```diff
@@ -46,6 +46,19 @@ If you are on a cluster you must edit the section [zworkers] and the parameter
 `host_cores`, replacing the `-1` with the number of cores to be used on
 each machine.
 
+### Help! I am running out of memory!
+
+If you are on a laptop, the first thing to do is to close the browser.
+If that is not enough, then close all other memory consuming
+applications. You can also limit the number of parallel threads as explained
+before (i.e. disable hyperthreading, reduce num_cores) or disable
```

Comment:
It is easy if you use the universal installer, however engine 3.11 uses the old installer. But since the final goal is to reduce the number of cores used we have an easy solution: setting concurrent_tasks=0 in the job.ini. There is no need to touch openquake.cfg.